model/configuration.py

```diff
@@ -733,7 +733,9 @@ def value(self):
 
     @value.setter
     def value(self, new_value):
-        if new_value is not None:
+        if isinstance(new_value, bytes):
+            new_value = new_value.decode("utf8")
```

Comment:
Presumably this is here because something was trying to store a `bytes` value into `ConfigurationSetting.value`, which is a Unicode column.
 
This is not generally safe and will raise a `UnicodeDecodeError` for invalid bytes. That said, it may be the case that all values destined to be stored as a  configurationsettings value maps cleanly to UTF-8.

I think the code that led to this solution is [already fixed here](https://github.com/ThePalaceProject/circulation/pull/19/files#diff-5cc8b5ac57b80db5ac78de8d4d0e3b2a074df1701374f4573d3284195121c927R533-R538).

That said, I see that other types (e.g., `int`s), are also converted to `str` in this setter, so I don't have a problem doing this for `bytes`.

Reply:
A difference between `bytes` and these other types (`int`, `float`,  JSON), is that `Configuration` has properties to reverse the conversion on the way out for the other types (e.g., `Configuration.float_val`).